extensions/amp-base-carousel/0.1/test-e2e/test-multi-visible-single-advance.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {getNextArrow, getPrevArrow, getSlides} from './helpers';
+import sleep from 'sleep-promise';
```

Comment:
nit: I personally prefer we'd write `sleep` ourselves instead of pulling in an npm lib. its basically a one liner :)

```js
function sleep(ms) {
  return new Promise(res => setTimeout(res, ms));
}
```